From 23e27de391858a6ed18510e53647e2030d3b293a Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sun, 22 Jun 2014 22:18:06 +0300
Subject: [PATCH] Look for libperl in multiarch paths

Starting with version 5.20.0 (currently in experimental), the Debian
perl package is moving libperl.so from /usr/lib to the multiarch
path /usr/lib/<triplet>, so instruct the configure script to look
there too.

The configure script will fall back to /usr/lib, so this is compatible
with both the old and the new setup.
---
 debian/rules | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index cf9d638..1e84922 100755
--- a/debian/rules
+++ b/debian/rules
@@ -7,13 +7,14 @@ else
   CFLAGS = -O1 -pedantic
 endif
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 build-stamp:
 	dh_testdir
 	./configure --prefix=/usr --mandir=/usr/share/man \
 	  --with-ssl \
 	  --with-ipv6 \
-	  --with-perl \
+	  --with-perl=/usr/lib/$(DEB_HOST_MULTIARCH) \
 	  --without-tcl \
 	  --program-transform-name='s/epic/epic4/' \
 	  --libexecdir=/usr/lib/epic4 --program-prefix=$(CURDIR)/debian/epic4
-- 
2.0.0

